docs/source/intro/troubleshooting.rst

```diff
@@ -77,6 +79,19 @@ A way to do it is to add a line similar to the following to the ``~/.bashrc`` an
 
 .. _Stackoverflow link: http://stackoverflow.com/questions/21079820/how-to-find-pg-config-pathlink
 
+
+.. _intro:troubleshooting:installation:quicksetup:
+
+Autodetection of the PostgreSQL setup
+.....................................
+
+Sometimes the local configuration of PostgreSQL may be difficult to autodetect for AiiDA when running ``verdi quicksetup``.
+In that case try the following:
+
+    1. Create the database in PostgreSQL (see :ref:`here<intro:install:database>`)
+    2. Run the full ``verdi setup`` command (see :ref:`here<intro:install:verdi_setup>`)
+
```

Comment:
Are users supposed to always do both steps or either one of them?

Reply:
Both of them; why? should I clarify this more in the previous text?

Reply:
Yes, I've made a suggestion above how to achieve that.